addons/website_sale/views/snippets/snippets.xml

```diff
@@ -18,6 +18,11 @@
                 <we-button data-set-ppr="3">3</we-button>
                 <we-button data-set-ppr="4">4</we-button>
             </we-select>
+            <we-select string="Default Sort" class="o_wsale_sort_submenu" data-no-preview="true">
+                <t t-foreach="request.env['website'].get_current_website().get_product_sort_mapping()" t-as="query_and_label">
```

Comment:
not need get_current_website() if it is static method
request.env['website'] will already return a browse record of type Website